README.md

```diff
@@ -0,0 +1,359 @@
+
+```
+Failed to cherry-pick 554626ada769abf82a5dabe6966afa4265acb6a6 into 2.7 :frowning_face:
+... Stopping here.
+
+To continue and resolve the conflict:
+    $ cherry_picker --status  # to find out which files need attention
+    # Fix the conflict
+    $ cherry_picker --status  # should now say 'all conflict fixed'
+    $ cherry_picker --continue
+
+To abort the cherry-pick and cleanup:
+    $ cherry_picker --abort
+```
+
+Passing the `--dry-run` option will cause the script to print out all the
+steps it would execute without actually executing any of them. For example:
+
+```console
+$ cherry_picker --dry-run --pr-remote pr 1e32a1be4a1705e34011770026cb64ada2d340b5 3.6 3.5
+Dry run requested, listing expected command sequence
+fetching upstream ...
+dry_run: git fetch origin
+Now backporting '1e32a1be4a1705e34011770026cb64ada2d340b5' into '3.6'
+dry_run: git checkout -b backport-1e32a1b-3.6 origin/3.6
+dry_run: git cherry-pick -x 1e32a1be4a1705e34011770026cb64ada2d340b5
+dry_run: git push pr backport-1e32a1b-3.6
+dry_run: Create new PR: https://github.com/python/cpython/compare/3.6...ncoghlan:backport-1e32a1b-3.6?expand=1
+dry_run: git checkout main
+dry_run: git branch -D backport-1e32a1b-3.6
+Now backporting '1e32a1be4a1705e34011770026cb64ada2d340b5' into '3.5'
+dry_run: git checkout -b backport-1e32a1b-3.5 origin/3.5
+dry_run: git cherry-pick -x 1e32a1be4a1705e34011770026cb64ada2d340b5
+dry_run: git push pr backport-1e32a1b-3.5
+dry_run: Create new PR: https://github.com/python/cpython/compare/3.5...ncoghlan:backport-1e32a1b-3.5?expand=1
+dry_run: git checkout main
+dry_run: git branch -D backport-1e32a1b-3.5
+```
+
+### `--pr-remote` option
+
+This will generate pull requests through a remote other than `origin`
+(e.g. `pr`)
+
+### `--upstream-remote` option
+
+This will generate branches from a remote other than `upstream`/`origin`
+(e.g. `python`)
+
+### `--status` option
+
+This will do `git status` for the CPython directory.
+
+### `--abort` option
+
+Cancels the current cherry-pick and cleans up the cherry-pick branch.
+
+### `--continue` option
+
+Continues the current cherry-pick, commits, pushes the current branch to
+`origin`, opens the PR page, and cleans up the branch.
+
+### `--no-push` option
+
+Changes won't be pushed to remote.  This allows you to test and make additional
+changes.  Once you're satisfied with local changes, use `--continue` to complete
+the backport, or `--abort` to cancel and clean up the branch.  You can also
+cherry-pick additional commits, by:
+
+```console
+$ git cherry-pick -x <commit_sha1>
+```
+
+### `--no-auto-pr` option
+
+PR creation page won't be automatically opened in the web browser or
+if GH_AUTH is set, the PR won't be automatically opened through API.
+This can be useful if your terminal is not capable of opening a useful web browser,
+or if you use cherry-picker with a different Git hosting than GitHub.
+
+### `--config-path` option
```

Comment:
This might look better as a list/table, rather than a series of titles+paragraph.

Reply:
Something like this?

In another branch: https://github.com/hugovk/cherry-picker/tree/convert-readme-to-md2

But compare with: 

https://github.com/hugovk/cherry-picker/tree/convert-readme-to-md?tab=readme-ov-file#--no-push-option

I used `<br><br>` to replicate a new paragraph, and we can only use single backticks around the command instead of three, so it's no longer copyable.